### ISSUE TITLE
There seems to be an issue with the error histogram in the code

### ISSUE THREAD

**Issue by HanLingsgjk** [lilygoli/SpotLessSplats#13]
Hello, I noticed that the error histogram in spotless_trainer.py looks strange while reading the code. I would like to ask you for advice
```
            info["err"] = torch.histogram(
                torch.mean(torch.abs(colors - pixels), dim=-3).clone().detach().cpu(),
                bins=cfg.bin_size,
                range=(0.0, 1.0),
            )[0]
```

the shape of 'torch.mean(torch.abs(colors - pixels), dim=-3).clone().detach().cpu()' is 1xWx3, it looks really strange, shouldn't it be 1xWxH？

**Comment by lilygoli** [comment]
Hi,
Thank you for pointing out the issue. The correct form is `dim=-1`, and the code has been updated accordingly.
I have verified the corrected form on `patio` and `patio_high` using `sls-mlp` to check if the results deviate significantly from the reported standard deviation in this re-implementation of SpotLessSplats. The results remain within the reported standard deviation:

patio:
PSNR: 22.647, SSIM: 0.8603, LPIPS: 0.057 Time: 0.006s/image Number of GS: 441674

patio_high:
PSNR: 23.199, SSIM: 0.8137, LPIPS: 0.151 Time: 0.004s/image Number of GS: 753848
